docker/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.6"
+
+x-omnia: &default
+  image: makerdao/omnia:latest
+  expose:
+    - ${SSB_NET_PORT}
+    - ${SSB_WS_PORT}
+  environment: &env
+    SSB_FILE_SET: /src/ssb-config.json
+    ETH_GAS: 7000000
+    SSB_SET_connections_incoming_net__external: '"${EXT_IP}"'
+    SSB_SET_connections_incoming_net__port: ${SSB_NET_PORT}
+    SSB_SET_connections_incoming_ws__external: '"${EXT_IP}"'
+    SSB_SET_connections_incoming_ws__port: ${SSB_WS_PORT}
+    SSB_SET_caps: ${SSB_CAPS}
+    OMNIA_SET_ethereum_from: '"${ETH_FROM}"'
+    OMNIA_SET_ethereum_keystore: '"/home/omnia/keys"'
+    OMNIA_SET_ethereum_password: '"/home/omnia/password"'
+    OMNIA_SET_ethereum_network: '"${ETH_NET-mainnet}"'
+    OMNIA_SET_ethereum_infuraKey: '"${INFURAKEY}"'
+    FILE_PATH_ETH_PASSWORD: /home/omnia/password
+    FILE_CONT_ETH_PASSWORD: ${ETH_PASSWORD}
+    FILE_PATH_ETH_KEY: /home/omnia/keys/key.json
+    FILE_CONT_ETH_KEY: ${ETH_KEY}
+
+services:
+  feed:
+    <<: *default
+    environment:
+      <<: *env
+      OMNIA_FILE_SET: /src/omnia/config/feed.conf
```

Comment:
I thought this what differentiated feed from relayer by passing this config in `OMNIA_FILE_SET` on each, but these files aren't getting populated by confer, `/home/omnia/omnia.conf` does though. Where do you differentiate feed from relayer then?

Reply:
You are right `OMNIA_SET` or `OMNIA_FILE_SET` determine what is the input JSON that is to be written to `/home/omnia/omnia.conf` which is the file that `omnia` will then read its config from.

If `OMNIA_SET` is set then the value of it will be used as a literal. Otherwise if `OMNIA_FILE_SET` is set then the value of it will be used as a file path inside the container and the content of that file will be used as the input.

The input JSON will then be modified by each `OMNIA_SET_*` and then written to the output (`/home/omnia/omnia.conf`). Because the `omnia` source code is copied into the docker image we can use `OMNIA_FILE_SET` to point to one of the default `/src/omnia/config/*.conf` files as input.

Reply:
Yep this makes sense